tcms/telemetry/api.py

```diff
@@ -131,6 +131,8 @@ def execution_trends(query=None):
     for status in TestExecutionStatus.objects.all():
         data_set[status.name] = []
         colors.append(status.color)
+    data_set['TOTAL'] = []
```

Comment:
string `TOTAL` should be marked for translation

Reply:
the logic for hiding the TOTAL data point depends on the css class `c3-target-TOTAL`, so if we translate total we will break that.

Reply:
It must be translated. 

The easiest workaround I can think about is rendering the translation of "TOTAL" as a data attribute in the html and then use the value as a jQuery selector and apply a well known class name to the legend elements that you want to hide.  This seems like the smallest possible change.

Another option is to use status IDs for the data points (b/c the legend class names use the translated names) but then I don't know how you would pass the actual strings around. 

A 3rd option is to calculate the total in JavaScript but you still need the translated string value for display purposes. 

A 4th option is to find a string/FA icon which doesn't need translation and can be universally understood as "TOTAL SUM" but it does make the code a bit uglier.

Reply:
1st one as easy enough to implement and it seems to do the job